### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by BradleyLi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+## Requirements
+- iOS 18.0 or later
+- Xcode 16.0 or later
+- Firebase account for backend services
+
+## Installation
+
+1. Clone the repository
+```bash
+git clone https://github.com/yourusername/FlatMate.git
```
Comment (Anidion) [inline README.md]:
```suggestion
git clone https://github.com/anidion/FlatMates.git
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+```
+
+2. Install dependencies
+```bash
+cd FlatMate
```
Comment (Anidion) [inline README.md:59]:
```suggestion
cd FlatMates
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
+# FlatMate - iOS Roommate Matching App
```
Comment (Anidion) [inline README.md]:
```suggestion
# FlatMates - iOS Roommate Matching App
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+FlatMate is a modern iOS application designed to help users find compatible roommates through a streamlined matching process.
```
Comment (Anidion) [inline README.md]:
```suggestion
FlatMates is a modern iOS application designed to help users find compatible roommates through a streamlined matching process.
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+5. Build and run the project in Xcode
+
+## License
+
+This project is licensed under the GNU Affero General Public License v3.0 - see the [LICENSE](LICENSE) file for details.
+
+## Contributing
+
+We welcome contributions to FlatMate! Please feel free to submit issues and pull requests.
```
Comment (Anidion) [inline README.md]:
```suggestion
We welcome contributions to FlatMates! Please feel free to submit issues and pull requests.
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+4. Push to the branch (`git push origin feature/AmazingFeature`)
+5. Open a Pull Request
+
+## Contact
+
+Project Link: [https://github.com/users/Anidion/projects/1/views/1](https://github.com/users/Anidion/projects/1/views/1)
+
+---
+
+Developed with ❤️ by Bradley Li and team.
```
Comment (Anidion) [inline README.md]: Let's put all our names in

**Review by Anidion — APPROVED** [review]
Looks good overall, just need to use the correct name mainly, and some other small fixes

File: README.md (hunk 6 above)
Comment (BradleyLi) [reply README.md]: Sorry, it is a mistake.

File: README.md (hunk 2 above)
Comment (BradleyLi) [reply README.md:59]: The folder name is actually called FlatMate lol

**Pull request merged**